src/main/java/com/example/backendspringboottechiteasycontroller/controllers/TelevisionController.java

```diff
@@ -0,0 +1,64 @@
+package com.example.backendspringboottechiteasycontroller.controllers;
+
+import com.example.backendspringboottechiteasycontroller.exceptions.RecordNotFoundException;
+import com.example.backendspringboottechiteasycontroller.service.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    private List<Television> televisions;
+
+    public TelevisionController(){
+        televisions = new ArrayList<>();
+        Television tv = new Television();
+        tv.setBrandName("test1");
+        tv.setBrandName("test2");
+
+    }
```

Comment:
Eigenlijk je je al die logic niet in je controller hebben. Als je een fake Db wilt maken doe dan:
` public List<Television> TELEVISION_ARRAYLIST = Arrays.asList(
            new Television("Phillips"),
            new Television("LG")
    );`